templates/migrate/phobos_02_create_failures.rb.erb

```diff
@@ -0,0 +1,23 @@
+class Phobos02CreateFailures < ActiveRecord::Migration[<%= ActiveRecord::Migration.current_version %>]
+  def up
+    create_table :failures do |t|
+      t.timestamp :created_at,     index: false
+      t.string    :topic,          index: true
+      t.string    :group_id,       index: true
+      t.string    :entity_id,      index: true
+      t.timestamp :event_time,     index: true
+      t.string    :event_type,     index: true
+      t.string    :event_version,  index: true
+      t.string    :checksum,       index: true
+      t.json      :payload,        index: false
+      t.json      :metadata,       index: false
+      t.string    :error_class,    index: false
+      t.string    :error_message,  index: false
+      t.json      :error_backtrace
+    end
+  end
```

Comment:
Douh! https://4.bp.blogspot.com/_CdjH-c4g0mI/ST6EUU_aFII/AAAAAAAAAnY/QFROKWIc5-M/s320/doh_l-761608.gif

Missed the bunch `index: true` on the migration... forget about htis.. carry on =)